packages/app/backend-http-client/src/client/constants.ts

```diff
@@ -0,0 +1,22 @@
+import type { MayOmit } from '@lokalise/node-core'
+import type { Client } from 'undici'
+
+import type { RequestOptions } from './types'
```

Comment:
`httpClient.ts` is already quite big and difficult to read, so extracting constants here

Reply:
great idea!